examples/material_gallery/lib/demo/menu_demo.dart

```diff
@@ -94,7 +94,7 @@ class MenuDemoState extends State<MenuDemo> {
                   child: new Text('Context menu item one')
                 ),
                 new PopupMenuItem(
-                  disabled: true,
+                  isDisabled: true,
```

Comment:
> maybe value != null?

I have mixed feelings about this kind of overloading since developers are unlikely to guess that it's how one disables an item. But if you really think it's a good idea, OK.